src/tools/sssctl/sssctl.c

```diff
@@ -305,6 +329,9 @@ int main(int argc, const char **argv)
         SSS_TOOL_DELIMITER("Certificate related tools:"),
         SSS_TOOL_COMMAND("cert-show", "Print information about the certificate", 0, sssctl_cert_show),
         SSS_TOOL_COMMAND("cert-map", "Show users mapped to the certificate", 0, sssctl_cert_map),
+#ifdef BUILD_PASSKEY
```

Comment:
Hi,

I think it would be good the have a new section like `Call SSSD's helper utilities` or `Passkey related tools` if there are plans to add more passkey related sub-commands.

bye,
Sumit

Reply:
I added
~~~
SSS_TOOL_DELIMITER("Passkey related tools:"),
~~~